### PR TITLE
Bump to 0.3

### PR DESCRIPTION
**Pull request by houli**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,10 +2,10 @@ version: '2'
...
 services:
   asclepius:
-    image: tomtoothfairies/asclepius:0.2
+    image: tomtoothfairies/asclepius:0.3
 
   panacea:
-    image: tomtoothfairies/panacea:0.2
+    image: tomtoothfairies/panacea:0.3
     ports:
       - '4000:4000'
```
Comment (c-brenn) [inline docker-compose.yml:11]: should chiron be tagged as well? it'll probably never be updated but just in case?

**Review by c-brenn — CHANGES_REQUESTED** [review]
looks good, just one question

File: docker-compose.yml (hunk 1 above)
Comment (houli) [reply docker-compose.yml:11]: Think we're ok with this tbh

Comment (c-brenn) [reply docker-compose.yml:11]: assume you need to add it to docker hub to build too

**Review by c-brenn — APPROVED** [review]

**Pull request merged**